packages-exp/auth-exp/src/core/providers/phone.ts

```diff
@@ -41,7 +42,11 @@ export class PhoneAuthProvider implements externs.PhoneAuthProvider {
     applicationVerifier: externs.ApplicationVerifier
```

Comment:
This should be
`verifyPhoneNumber(phoneNumber: externs.PhoneInfoOptions | string, ...`

Reply:
done